openapi.yaml

```diff
@@ -447,6 +447,15 @@ components:
           example: JWT
           type: string
           enum: [JWT]
+    TopicRequestTypes:
+      title: Type of request related to topic
+      type: string
+      enum:
```

Comment:
There are several ways to write `enum` (and `string`, with optional quotes, and probably other things) in yaml. I'm thinking we should have a linter to enforce consistency, because there is already discrepancies in styles :/

Reply:
Good point. I should have checked how we did that in the past. I followed one example from the docs without thinking too much about it. I'll add a housekeeping ticket for looking up linter rules for that, is that ok?

Reply:
Yes, perfect ^^ As an example, here is a different enum haha: https://github.com/aiven/klaw/blob/f67a5fd7d9cecd3cf40633e971a3c6ef9c388480/openapi.yaml#L105

Reply:
added a ticket! https://github.com/aiven/klaw/issues/417